src/fragments/lib/auth/common/mfa/flows.mdx

```diff
@@ -251,6 +310,42 @@ Future<void> confirmTotpUser(String totpCode) async {
 After a user has been signed in, call `updateMFAPreference` to record the MFA type as enabled for the user
 and optionally set it as preferred so that subsequent logins default to using this MFA type.
 
+<InlineFilter filters={['android']}>
```

Comment:
Also can we reuse snippet [src/fragments/lib/auth/android/mfa_preference/20_update_mfa_preference.mdx](https://github.com/aws-amplify/docs/pull/5828/files#diff-2e3935b4591c0025c39f7cc7718bc4ec901f81fca6723da73d716fed3342c94f) here?